hnn_core/params.py

```diff
@@ -662,9 +662,79 @@ def compare_dictionaries(d1, d2):
     return d1
 
 
+def _any_positive_weights(drive):
+    """ Checks a drive for any positive weights. """
+    weights = (list(drive['weights_ampa'].values()) +
+               list(drive['weights_nmda'].values()))
+    if any([val > 0 for val in weights]):
+        return True
+    else:
+        return False
+
+
+def _get_n_cells(net):
+    """ Get number of neocortical cells  """
+    cell_names = net.cell_types.keys()
+    n_cells = max([net.gid_ranges[cell_name].stop for cell_name in cell_names])
+    return n_cells
```

Comment:
There's technically and `_n_cells` attribute that I believe is updated separately for `_n_gids`.

Do you think it's better to use the attribute or calculate the number of cells directly?

Reply:
Ah thanks for pointing this out. The function isn't used anymore so I deleted it.